tests/basic/storage_pool_test.go

```diff
@@ -8828,3 +8829,266 @@ var _ = Describe("{ReplResyncOnPoolExpand}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+// Volume replication change
+var _ = Describe("{VolumeHAPoolOpsNoKVDBleaderDown}", func() {
+	var testrailID = 0
+	// JIRA ID :https://portworx.atlassian.net/browse/PTX-17728
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("VolumeHAPoolOpsNoKVDBleaderDown",
+			"Test Volume HA Pool Operations should not make KVDB node down", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+	stepLog := "has to schedule apps and update replication factor for attached node"
+	It(stepLog, func() {
+		var wg sync.WaitGroup
+		numGoroutines := 3
+
+		wg.Add(numGoroutines)
+		done := make(chan bool)
+		errChan := make(chan error)
+
+		volumesCreated := []string{}
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("volumepooloperations-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get Pool with running IO on the cluster
+		poolUUID, err := GetPoolIDWithIOs(contexts)
+		log.FailOnError(err, "Failed to get pool running with IO")
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+
+		terminate := false
+		stopRoutine := func() {
+			if !terminate {
+				done <- true
+				wg.Done()
+				close(done)
```

Comment:
please use defer GinkgoRecover() to handle panic in the routines

Reply:
done